website/docs/docs/cloud/manage-access/self-service-permissions.md

```diff
@@ -1,42 +1,81 @@
 ---
-title: "Self-service permissions"
-description: "Learn how dbt Cloud administrators can use self-service permissions to control access in a dbt Cloud account."
+title: "Self-service Team account permissions"
+description: "Learn how dbt Cloud administrators can use self-service permissions to control access in a dbt Cloud Team account."
+sidebar_label: "Team permissions"
 id: "self-service-permissions"
 ---
 
-import Permissions from '/snippets/_self-service-permissions-table.md';
+Self-service Team accounts are a quick and easy way to get dbt Cloud up and running for a small team. Team accounts have limited access to features and advanced settings like SSO, group management, and larger user bases will require an [Enterprise](/docs/cloud/manage-access/enterprise-permissions) account. If you're interested in upgrading, contact [dbt Labs today](https://www.getdbt.com/contact)!
 
+## Groups and permissions
 
-<Permissions features={'/snippets/_self-service-permissions-table.md'}/>
+Groups determine a users permission and there are three groups are available for Team plan dbt Cloud accounts: Owner, Member, and Everyone. The first Owner user is the person who created the dbt Cloud account. New users are added to the Member and Everyone groups when they onboardbut this can be changed when the invitation is created. These groups only affect users with a [Developer license](#licenses) assigned. 
```

Comment:
```suggestion
Groups determine a user's permission and there are three groups are available for Team plan dbt Cloud accounts: Owner, Member, and Everyone. The first Owner user is the person who created the dbt Cloud account. 

New users are added to the Member and Everyone groups when they onboard but this can be changed when the invitation is created. These groups only affect users with a [Developer license](#licenses) assigned. 
```